apps/custom-trigger/src/routes/subscription.js

```diff
@@ -0,0 +1,12 @@
+// This file defines which functions are invoked when a particular route/endpoint is called.
+
+const express = require('express');
+const router = express.Router();
+
+const subscriptionController = require('../controllers/subscription-controller.js');
+const authenticationMiddleware = require('../middlewares/authentication').authenticationMiddleware;
+
+router.post('/subscriptions/subscribe', subscriptionController.subscribe);
```

Comment:
We need to add authneticationMiddleware to routes, like here: https://github.com/mondaycom/welcome-apps/blob/master/apps/quickstart-integrations/src/routes/transformation.js#L7